### DISCUSSION TITLE
How to show Manager button for ComfyUI

### DISCUSSION THREAD

**Discussion by SyamsQ** [Acly/krita-ai-diffusion#736] [Q&A] [answered]
I'm trying to install Supir via ComfyUI Krita. In many tutorials, it is necessary to click the Manager button. But in ComfyUI on my computer there is no Manager button.
How do I make that button appear?
Is it possible to install Upscaler Supir for Krita?
![image](https://github.com/Acly/krita-ai-diffusion/assets/17028491/780304c0-ba88-4fb9-b990-d5401bf978b3)

**Comment by Acly** [comment, marked as answer]
> How do I make that button appear?

You need to install https://github.com/ltdrdata/ComfyUI-Manager
It's not part of base ComfyUI, and not required or installed by the Krita Plugin

> Is it possible to install Upscaler Supir for Krita?

No it's not supported by the plugin. You can use it externally via ComfyUI directly.

**Comment by SyamsQ** [comment]
doesn't the Kirta plugin already have Comfyui installed, if I install comfyui externally, that means there are two comfyui on one PC?

**Reply by Acly** [reply]
It's up to you:
* Use only Krita plugin managed ComfyUI, you will not be able to use SUPIR
* Have two ComfyUI (one for Krita plugin, one custom). You can share models to save space
* Only use custom external ComfyUI. The Krita plugin works with external ComfyUI too (use the Connect option), provided it has all the requirements installed

**Reply by SyamsQ** [reply]
Are you sure, they can share models?

**Reply by moonwhaler** [reply]
Softlinking, I suppose.
